Specs/Scene/I3dmParserSpec.js

```diff
@@ -0,0 +1,45 @@
+import { I3dmParser } from "../../Source/Cesium.js";
+import Cesium3DTilesTester from "../Cesium3DTilesTester.js";
+import createScene from "../createScene.js";
+
+describe(
+  "Scene/I3dmParser",
+  function () {
+    var scene;
+
+    beforeAll(function () {
+      scene = createScene();
+
+      // Keep the error from logging to the console when running tests
+      spyOn(I3dmParser, "_deprecationWarning");
+    });
+
+    afterAll(function () {
+      scene.destroyForSpecs();
+    });
+
+    afterEach(function () {
+      scene.primitives.removeAll();
+    });
+
+    it("throws with undefined arrayBuffer", function () {
+      expect(function () {
+        I3dmParser.parse(undefined);
+      }).toThrowDeveloperError();
+    });
+
+    it("throws with empty gltf", function () {
+      // Expect to throw error due to invalid glTF magic.
+      var arrayBuffer = Cesium3DTilesTester.generateInstancedTileBuffer();
+      Cesium3DTilesTester.loadTileExpectError(scene, arrayBuffer, "i3dm");
+    });
+
+    it("throws with unsupported I3DM version", function () {
```

Comment:
```suggestion
    it("throws with unsupported i3dm version", function () {
```

all the 3D Tiles 1.0 formats are lowercase. in my PR I fixed this wherever I saw uppercase B3DM, but we should update this one here too.